go/test/endtoend/onlineddl/scheduler/onlineddl_scheduler_test.go

```diff
@@ -2400,3 +2518,26 @@ func getCreateTableStatement(t *testing.T, tablet *cluster.Vttablet, tableName s
 	statement = queryResult.Rows[0][1].ToString()
 	return statement
 }
+
+func runInTransaction(t *testing.T, tablet *cluster.Vttablet, query string, commitTransactionChan chan any, transactionErrorChan chan error) error {
+	conn, err := tablet.VttabletProcess.TabletConn(keyspaceName, true)
+	require.NoError(t, err)
+	defer conn.Close()
+
+	_, err = conn.ExecuteFetch("begin", 0, false)
+	require.NoError(t, err)
+
+	_, err = conn.ExecuteFetch(query, 10000, false)
+	require.NoError(t, err)
+
+	if commitTransactionChan != nil {
+		// Wait for instruction to commit
+		<-commitTransactionChan
```

Comment:
I feel like it's worth passing in a context to terminate on timeout but not a big deal as it's a test.

Reply:
Agreed, done.